doc/getting_started/toolchain_3rd_party_x_compilers.rst

```diff
@@ -210,6 +210,84 @@ You can build toolchains from source code using crosstool-NG.
       $ echo $XTOOLS_TOOLCHAIN_PATH
       /Volumes/CrossToolNGNew/build/output/
 
+Cadence Tensilica Xtensa C/C++ Compiler (XCC)
+*********************************************
+
+#. Obtain Tensilica Software Development Toolkit targeting the specific SoC
```

Comment:
Other toolchains have a link on where to find it.
Would that be possible here ?

For example like this:
https://github.com/zephyrproject-rtos/zephyr/blob/85a1aecbfe0a8da2c5e0559341f748a8c6d0f42b/doc/getting_started/toolchain_3rd_party_x_compilers.rst#L119
https://github.com/zephyrproject-rtos/zephyr/blob/85a1aecbfe0a8da2c5e0559341f748a8c6d0f42b/doc/getting_started/toolchain_3rd_party_x_compilers.rst#L144
https://github.com/zephyrproject-rtos/zephyr/blob/85a1aecbfe0a8da2c5e0559341f748a8c6d0f42b/doc/getting_started/toolchain_3rd_party_x_compilers.rst#L291-L293

Reply:
Unfortunately, no. One has to talk to Cadence to obtain the SDK.